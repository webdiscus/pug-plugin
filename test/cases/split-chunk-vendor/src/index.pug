html
  head
    link(href=require('./style.scss') rel='stylesheet')
    script(src=require('@test-fixtures/lorem') defer)
  body
    h1 Hello Pug!
    script(src=require('./main'))