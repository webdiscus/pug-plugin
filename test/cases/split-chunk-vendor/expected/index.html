<!DOCTYPE html><html><head><link href="/assets/css/style.026fd625.css" rel="stylesheet"><script src="/assets/js/runtime.a5bd90e1.js" defer></script><script src="/assets/js/vendors.9326a07a.js" defer></script><script src="/assets/js/fixture-lorem.b2ee76da.js" defer></script></head><body><h1>Hello Pug!</h1><script src="/assets/js/main.3ca131f0.js"></script></body></html>